lib/store/index.js

```diff
@@ -7,12 +7,25 @@
 import { createStore, applyMiddleware } from 'redux';
 import reducer from './reducer';
 import thunk from 'redux-thunk';
+import createLogger from 'redux-logger';
```

Comment:
Oh man, that really sucks!

I don't want to import code if I'm not going to use it later :( ... some feedback for the spec.